docs/fastdds/env_vars/env_vars.rst

```diff
@@ -252,6 +252,40 @@ The following example shows how to set the address of two remote discovery serve
     (that has been initialized with this environment variable previously) if loaded from an environment file using
     :ref:`env_vars_fastdds_environment_file`.
 
+.. _env_vars_easy_mode:
+
+``EASY_MODE`` (Beta)
+--------------------
+
+.. warning::
+    Discovery Server ``EASY_MODE`` is a Beta feature. Its functionalities are still under development and they might contain bugs.
+    Its final version will be available soon. Any feedback is appreciated and can be provided at:
```

Comment:
Perhaps it would be a good idea moving the warning at the end to this one stating that the feature is not available in windows